script.io.benchmark/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 thodnev
```

Comment:
I would think this should be your nick. But this line should be in `monkeytest.py`
Also add your nick to default.py